tests/integration-service/integration.go

```diff
@@ -212,7 +212,7 @@ var _ = framework.IntegrationServiceSuiteDescribe("Integration Service E2E tests
 
 			env, err = f.AsKubeAdmin.GitOpsController.CreatePocEnvironment(EnvironmentName, testNamespace)
 			Expect(err).ShouldNot(HaveOccurred())
-			integrationTestScenario, err = f.AsKubeAdmin.IntegrationController.CreateIntegrationTestScenario(applicationName, testNamespace, BundleURLFail, InPipelineNameFail)
```

Comment:
can we also remove the now-unused variables like: `BundleURL`, `BundleURLFail`, etc?

Reply:
Removed all unused consts.

Reply:
Thanks!